templates/web/brent/report/new/roads_message.html

```diff
@@ -14,3 +14,11 @@
         <p><strong>Please select a park or highway from the map</strong></p>
         <p>To report this issue please select a verge or shrub bed that is in a Brent maintained park or within the boundaries of roads and pavements the council is responsible for</p>
     </div>
+    <div id="js-brent-allotments" class="hidden js-responsibility-message">
```

Comment:
Don't think I've emailed about this yet, but recently I added the ability to have this message functionality work from config - `no_asset_message`, then we don't need to update this template. (Rest of code still needed/same.) - Brent is only user so far, as an example.